SECP-service/src/main/java/com/visucius/secp/models/User.java

```diff
@@ -9,16 +9,27 @@
 
 @Entity
 @Table(name = "users")
+@NamedQueries(
+    {
+        @NamedQuery(
+            name = "com.visucius.secp.models.User.findByUsername",
```

Comment:
can you adjust the settings in your intellij, you should not be using the full path com.visucius.secp.models.User here. It should be imported automatically.

Reply:
This has nothing to do with the settings in intellij,  you need to pass the name of the query you specified in the user object. I can change the name to be findByUsername instead.

Reply:
Exactly what I mean. I don’t see a point of putting the whole path as a namedquery key. A single word as name is enough.